samples/model-builder/model_registry/upload_new_model_version_using_custom_training_pipeline_sample.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START aiplatform_model_registry_upload_new_model_version_using_custom_training_pipeline]
+
+from typing import List
+
+from google.cloud import aiplatform
+
+
+def upload_new_model_version_using_custom_training_pipeline(
+    display_name: str,
+    script_path: str,
+    container_uri,
+    model_serving_container_image_uri: str,
+    dataset_id: str,
+    replica_count: int,
+    machine_type: str,
+    accelerator_type: str,
+    accelerator_count: int,
+    model_id: str,
+    args: List[str],
+    model_version_aliases: List[str],
+    model_version_description: str,
+    is_default_version: bool,
+    project: str,
+    location: str,
+):
+    """
+    Uploads a new model version using a custom training pipeline.
+    Args:
+        display_name: The display name of the model version.
+        script_path: The path to the Python script that trains the model.
+        container_uri: The URI of the container to use for training.
+        model_serving_container_image_uri: The URI of the serving container image to use.
+        dataset_id: The ID of the dataset to use for training.
+        replica_count: The number of replicas to use for training.
+        machine_type: The machine type to use for training.
+        accelerator_type: The accelerator type to use for training.
+        accelerator_count: The number of accelerators to use for training.
+        model_id: The ID of the model to upload a new version to. Parent resource name of the model is also accepted.
+        args: A list of arguments to pass to the training script.
+        model_version_aliases: The aliases of the model version.
+        model_version_description: The description of the model version.
+        is_default_version: Whether the model version is the default version.
```

Comment:
ditto

Reply:
I highlighted that the model aliases are related to the model to create